api/src/main/java/org/openmrs/module/pihcore/setup/LocationTagSetup.java

```diff
@@ -174,7 +174,7 @@ private static void setupLocationTagsForLiberia(LocationService locationService)
         setLocationTagsFor(locationService, LocationTags.VISIT_LOCATION, Arrays.asList(LiberiaLocations.HEALTH_FACILITY));
 
         List<LocationDescriptor> liberiaFacilityLocations = Arrays.asList(LiberiaLocations.RECORDS_ROOM,
-                LiberiaLocations.OUTPATIENT_CLINIC, LiberiaLocations.MENTAL_HEALTH_CLINIC, LiberiaLocations.NCD_CLINIC, LiberiaLocations.MCH_CLINIC);
+                LiberiaLocations.OUTPATIENT_CLINIC, LiberiaLocations.MENTAL_HEALTH_CLINIC, LiberiaLocations.NCD_CLINIC, LiberiaLocations.MCH_CLINIC, LiberiaLocations.EMERGENCY);
```

Comment:
Sorry @BranfordTGbieor , in my comment yesterday I had meant that you should submit this change _after_ my refactor, I didn't think you were making the change so soon. You'll need to assign the location tags in the [pih config json](https://github.com/PIH/openmrs-config-pihliberia/blob/7ed939cd9e865ffe89af3e1afa8f73467bdc439b/configuration/pih/pih-config-liberia.json#L15) file. This function here is being deleted.

Reply:
Ok @brandones, I'll do that and issue the PR.